Source/WebCore/rendering/ios/RenderThemeIOS.mm

```diff
@@ -600,6 +597,16 @@ static void adjustInputElementButtonStyle(RenderStyle& style, const HTMLInputEle
     context.fillPath(glyphPath);
 }
 
+bool RenderThemeIOS::paintMenuListButton(const RenderObject& box, const PaintInfo& paintInfo, const IntRect& rect)
+{
+#if ENABLE(MAC_STYLE_CONTROLS_ON_CATALYST)
+    if (paintMenuListButtonForVectorBasedControls(box, paintInfo, rect))
+        return false;
```

Comment:
hmm we've normally been checking the setting in the condition and then `return RenderThemeCocoa::` inside.